Js/imageLoader.js

```diff
@@ -1,5 +1,15 @@
+/**
+ adds image i for each div there are in the html file
+ */
 $(document).ready( function() {
     $("div").each( function(i) {
-        $(this).append("<img src='Media/"+(++i)+".png' width='600' height='600' />");
+        $(this).append("<img src='Media/Photos/"+(++i)+".png' width='630' height='600'/>");

```

Comment:
The server does not separate between photo and videos.
Instead, look at the file extension of each file in `Media/` if it is `.png` or `.mp4`.

Reply:
I changed it but I don't know if it showed up here